### PR TITLE
Add the fixed 2.2.1 and update rolling versions

### LINKED ISSUE #115: Add the 2.2.1 tag and update the other tags to the latest Tarantool  versions

### PR DESCRIPTION
**Pull request by avtikhon**
Added fixed version 2.2.1.
Updated rolling versions:
  1.7: 1.7.6-11-gcd17b77f9 -> 1.7.6-27-g7ef5be2ee
  1.10/1.x-centos7: 1.10.3-93-gdf8f1fdaa -> 1.10.3-107-g17db27170
  2.1: 2.1.2-119-gc118d5142 -> 2.1.2-143-g3edaaed6c
  2.x-centos7: 2.1.2-119-gc118d5142 -> 2.2.1-3-g878e2a42c
  2.2: created with 2.2.1-3-g878e2a42c
  2.x: 2.2.0-482-g8c84932ad -> 2.3.0-1-g35ef33200
    
Closes #115

### CODE REVIEW

File: 2.x-centos7/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ MAINTAINER mail@racktear.com
 RUN groupadd tarantool \
     && adduser -g tarantool tarantool
 
-ENV TARANTOOL_VERSION=2.1.2-119-gc118d5142 \
+ENV TARANTOOL_VERSION=2.1.2-143-g3edaaed6c \
```
Comment (olegrok) [inline 2.x-centos7/Dockerfile]: Please, update this image to 2.2

**Comment by Totktonada** [comment]
I doubt we should split 2.1 and 2.2.

**Comment by Totktonada** [comment]
Typo: 1.1 → 1.10 (commit message).

**Comment by avtikhon** [comment]
Rechecked the new version for 1.6 and failed to build - removed its update.
Corrected commit message for 1.10.
Updated 2.x-centos7 to the latest 2.2 version 2.2.1-3-g878e2a42c.

File: 2.x-centos7/Dockerfile (hunk 1 above)
Comment (avtikhon) [reply 2.x-centos7/Dockerfile]: Updated 2.x-centos7 to the latest 2.2 version 2.2.1-3-g878e2a42c. Is it Ok ?

Comment (olegrok) [reply 2.x-centos7/Dockerfile]: Yes, thanks

File: README.md [hunk 2]
```diff
@@ -337,6 +338,7 @@ Rolling versions:
 | master | 1.x/Dockerfile | 1          |
 | master | 1.x/Dockerfile | latest     |
 | master | 2.1/Dockerfile | 2.1        |
+| master | 2.2/Dockerfile | 2.2        |
```
Comment (Totktonada) [inline README.md]: Now I see, we have rolling 2.1 and it worth to add 2.2 too. Okay.

File: 2.x-centos7/Dockerfile [hunk 3]
```diff
@@ -4,7 +4,7 @@ MAINTAINER mail@racktear.com
 RUN groupadd tarantool \
     && adduser -g tarantool tarantool
 
-ENV TARANTOOL_VERSION=2.1.2-119-gc118d5142 \
+ENV TARANTOOL_VERSION=2.2.1-3-g878e2a42c \
```
Comment (Totktonada) [inline 2.x-centos7/Dockerfile]: Why tarantool version in 2.x-centos7 is not the same as in 2.x? I don't get your approach to choose a version.

**Pull request merged**